README.md

```diff
@@ -12,7 +12,7 @@
 - Configuration handling 
 - Pluggable agents
 
-Example to implement a monitoring agent:
+## Example to implement a monitoring agent:
```

Comment:
For sending customer metrics?

Reply:
For all metrics. There is no difference anymore.